parquet-hadoop/src/main/java/org/apache/parquet/hadoop/RowGroupMerger.java

```diff
@@ -0,0 +1,634 @@
+/*
```

Comment:
If this file is only used by tools, can we move it out of parquet-hadoop project?